server/middlewares/validation.js

```diff
@@ -0,0 +1,60 @@
+import Joi from '@hapi/joi';
+
+function validator(req, res, schema, next) {
```

Comment:
Hey @alainmateso use ES6 for this function

Reply:
Done